hal/halx86/apic/apictrap.S

```diff
@@ -20,7 +20,7 @@ TRAP_ENTRY HalpProfileInterrupt, (TF_VOLATILES OR TF_SEND_EOI)
 
 PUBLIC ApicSpuriousService
 ApicSpuriousService:
-    iret
```

Comment:
Just curious, shouldn't `iret` do the same thing as `iretq` when compiled as amd64?

Reply:
See https://www.felixcloutier.com/x86/iret:iretd